demisto_sdk/tests/integration_tests/generate_docs_integration_test.py

```diff
@@ -0,0 +1,95 @@
+import pytest
+from demisto_sdk.commands.common.git_tools import git_path
+from os.path import join
+from pathlib import Path
+
+from click.testing import CliRunner
+from demisto_sdk.__main__ import main
+
+GENERATE_DOCS_CMD = "generate-docs"
+DEMISTO_SDK_PATH = join(git_path(), "demisto_sdk")
+VALID_PLAYBOOK_WITH_IO = join(DEMISTO_SDK_PATH, "tests/test_files/playbook-Test_playbook.yml")
```

Comment:
these seem to be used only in specific tests, why do they need to be global consts?